packages/compiler-cli/src/ngtsc/core/src/compiler.ts

```diff
@@ -191,6 +191,12 @@ export class NgCompiler {
       }
     }
 
+    this.diagnostics.forEach(diag => {
+      if (diag.code && HAS_GUIDE.has(ngErrorCode(diag.code))) {
+        diag.messageText += `. Find more at https://angular.io/errors/NG${ngErrorCode(diag.code)}`;
```

Comment:
I'd propose extracting error page base URL to a separate const (for both runtime and compiler files), so it's easier to manage it in case the base path should be different inside g3.

```suggestion
// Base URL for the error details page.
// Keep this value in sync with a similar const in `packages/core/src/render3/error_code.ts`.
const ERROR_DETAILS_PAGE_BASE_URL = 'https://angular.io/errors';
...
        diag.messageText += `. Find more at ${ERROR_DETAILS_PAGE_BASE_URL}/NG${ngErrorCode(diag.code)}`;
```